### DISCUSSION TITLE
What to do with lxqt-themes-extra?

### DISCUSSION THREAD

**Discussion by stefonarch** [lxqt/lxqt#1899] [Ideas]
I went in the basements of the 200* issues today and stumpled also upon this empty repo https://github.com/lxqt/lxqt-themes-extra and this discussion https://github.com/lxqt/lxqt/issues/721

In the wiki are linked some themes: https://github.com/lxqt/lxqt/wiki/Themes

As an idea I would like that packagers could create a lxqt-themes-extra package.

There are a few points:

1) which themes (from the wiki or other sources) are suitable. Testing, criteria,"quality", deciding and so on.
2) License and credit stuff (it cannot be displayed in the GUI nowhere atm)
3) others that doesn't come to my mind

**Comment by tsujan** [comment]
IMHO, lxqt-themes-extra is redundant. Theme PRs can be made against lxqt-themes; they'll be merged if they're acceptable. If they aren't acceptable, why adding them elsewhere?

**Reply by stefonarch** [reply]
The difference is that not everybody will make a PR for different reasons, some aren't even on GH. 
It would be up to us to collect them.

**Reply by tsujan** [reply]
Why should we make an exception for it?

More importantly, if PRs aren't made, how could we review them? Everything that comes into LXQt needs to come through PRs; otherwise, a mess will be created.
